incomplete patch.py

```diff
@@ -96,14 +96,16 @@ def eat():
 
 def new_food():
 	display()
-	food[0]=random.randint(0,1000-pixel)
-	food[1]=random.randint(0,800-pixel)
+	food[0]=random.randint(0+pixel,1000-pixel)
+	food[1]=random.randint(0+pixel,800-pixel)
 	pygame.draw.circle(screen, YELLOW, (food[0], food[1]), pixel)
 	pygame.display.update()
 	pass
 
 screen = pygame.display.set_mode(size)
 clock = pygame.time.Clock()
+timer_event = pygame.USEREVENT + 1
```

Comment:
i don't think this is necessary since i added threading to the program and it takes care of that problem